CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # NVIDIA MIG Manager Changelog
 
+## v0.11.0
+- Fix rpm spec to maintain the config.yaml and hooks.yaml symlinks during an update
+- Update the .gitignore to skip the dist directory.
```

Comment:
nit: can we remove the version heading since we have not bumped the project version in `versions.mk` yet? Also, I would prefer to remove the second bullet about `.gitignore` from the changelog.
```suggestion
- Fix rpm spec to maintain the config.yaml and hooks.yaml symlinks during an update
```